packages/mint-components/CHANGELOG.md

```diff
@@ -374,8 +386,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - \<sqm-stencilbook>
 
 [unreleased]: https://github.com/saasquatch/program-tools/compare/mint-components@1.5.1...HEAD
```

Comment:
This needs to be updated for the latest version 1.5.2...HEAD

Reply:
👍 fixed